Userland/Utilities/cut.cpp

```diff
@@ -140,20 +140,26 @@ static void process_line_bytes(StringView line, Vector<Range> const& ranges)
     outln();
 }
 
-static void process_line_characters(StringView line, Vector<Range> const& ranges)
+static ErrorOr<void> process_line_characters(StringView line, Vector<Range> const& ranges)
 {
     for (auto const& range : ranges) {
         if (range.m_from > line.length())
             continue;
 
-        auto s = String::from_utf8(line).release_value_but_fixme_should_propagate_errors();
```

Comment:
Does just `auto s = TRY(String::from_utf8(line));` not work here?

Reply:
You're right. Fixed.